packages/webpack/src/executors/webpack/schema.d.ts

```diff
@@ -64,22 +57,40 @@ export interface WebpackExecutorOptions {
   outputPath: string;
   poll?: number;
   polyfills?: string;
-  postcssConfig?: string;
   progress?: boolean;
   runtimeChunk?: boolean;
-  scripts?: Array<ExtraEntryPointClass | string>;
   sourceMap?: boolean | 'hidden';
   statsJson?: boolean;
-  stylePreprocessorOptions?: any;
-  styles?: Array<ExtraEntryPointClass | string>;
-  subresourceIntegrity?: boolean;
   target?: 'node' | 'web';
   transformers?: TransformerEntry[];
   tsConfig: string;
   vendorChunk?: boolean;
   verbose?: boolean;
   watch?: boolean;
   webpackConfig?: string;
+  // TODO(jack): Also deprecate these in schema.json once we have migration from executor options to webpack.config.js file.
```

Comment:
These are just reordered with new `deprecated comments.

Reply:
Is there a way to 'deep link' the deprecated message to the new function?